### PR TITLE
Login

### PR DESCRIPTION
**Pull request by parvez-rashid**
View the screenshot below, preview link has some problem.

image

### CODE REVIEW

File: login/index.html [hunk 1]
```diff
@@ -26,7 +26,9 @@
       </label>
       <ul class="list-style-none list-inline">
         <li>
-          <a class="options link" href="">profile</a>
+          <a class="options link link-active" href="/login/index.html"
+            >login/signup</a
+          >
         </li>
         <li>
           <a class="options link" href="">wishlist</a>
```
Comment (kaustubh-jsr) [inline login/index.html:34]: Just a suggestion, you can have href="#", this way it won't reload the page on clicking every time, as long as you don't have a working link

**Review by kaustubh-jsr — COMMENTED** [review]
The UI looks good, minor suggestion in the comments.Looks good to merge

**Review by eakansh125 — APPROVED** [review]
Could not find an issue, a little padding can be given inside the login box for better UI, but not a mandate. Good to be merged from my end.

**Comment by amantulsyan35** [comment]
apart from @eakansh125  and @kaustubh-jsr  cannot find any issues, good to merge

File: login/index.html (hunk 1 above)
Comment (parvez-rashid) [reply login/index.html:34]: Thanks for the suggestion, will include that.

**Comment by parvez-rashid** [comment]
> Could not find an issue, a little padding can be given inside the login box for better UI, but not a mandate. Good to be merged from my end.

Thanks for the suggestion, will try to adjust it.

**Pull request closed without merging**